command/server/init.go

```diff
@@ -66,14 +61,6 @@ func (p *serverParams) initRawParams() error {
 	return p.initAddresses()
 }
 
-func (p *serverParams) initBlockTime() error {
-	if p.rawConfig.BlockTime < 1 {
```

Comment:
Do we need to keep this condition somewhere? maybe in `extractBlockTime`?

Reply:
You are right, fixed: https://github.com/0xPolygon/polygon-edge/pull/1365/commits/2b0c446f29fb07fe86e6c2ea100ff409cbcac570